core/server/master/src/main/java/alluxio/master/metrics/DefaultMetricsMaster.java

```diff
@@ -89,22 +96,61 @@ public Object getValue() {
         });
   }
 
+  @VisibleForTesting
+  protected void addMultiValueAggregator(MultiValueMetricsAggregator aggregator) {
```

Comment:
Since the type is different, we could call these both `addAggregator`? Up to you

Reply:
Sure